pkgs/development/libraries/libfprint-2-tod1-goodix/default.nix

```
@@ -25,6 +25,7 @@ stdenv.mkDerivation {
   '';
 
   passthru.driverPath = "/lib/libfprint-2/tod-1";
+  passthru.needsLibfprint_1_90 = true;  # https://gitlab.freedesktop.org/libfprint/libfprint/-/issues/484
```

```suggestion
  passthru = {
    driverPath = "/lib/libfprint-2/tod-1";
    # https://gitlab.freedesktop.org/libfprint/libfprint/-/issues/484
    needsLibfprint_1_90 = true;
  };
```
same for the other 2